packages/ckeditor5-html-support/src/schemadefinitions.js

```diff
@@ -47,6 +47,9 @@
 // Skipped hidden elements:
 // noscript
 // script
+//
+// If you wish to add support for (no)script elements, please familiarize yourself with
+// comment https://github.com/cksource/ckeditor5-internal/issues/949#issuecomment-913521257 first.
```

Comment:
I'm not sure if that comment in the issue is still valid. This PR is introducing the script filter.

Reply:
Sure, will remove it. It seems like it's not valid anymore.